docs/platform/tutorials/image-processing/other-services.md

```diff
@@ -0,0 +1,195 @@
+# Other services
+
+There are some additional services in the pipeline that provide useful functionality. These range from S3 storage of data to calculation of the maximum vehicles per day in a specific location.
+
+![Other services](../image-processing/images/other-services-pipeline-segment.png)
+
+Briefly, these services are:
+
+1. *Cam vehicles* - calculates the total vehicles, where vehicle is defined as one of: car, 'bus', 'truck', 'motorbike'. This number is fed into the *Max vehicle window* service.
+
+2. *Max vehicle window* - calculates the maximum vehicles over a time window of one day. This service sends messages to the Data API service.
```

Comment:
Why is the "Data Buffer" service missing? This is the one that sits in between the 1) cam vehicles and the max vehicle window and 2) the front end

Reply:
Yes, the pipeline is now quite different to when I originally wrote the tutorial so it will probably have to change. :( However, if I remember correctly the data buffer was subsumed into one of the other services. I will have to go back to it and check at some point though.

The problem was there was lots of work being done on multiple branches (envs), and not all pushed up, and so quite hard to track what was going on. If you try to have too many envs you run into account restrictions too. 

When the project is "done" I will have to go back and make exactly these sorts of fixes. Same applies to your comment on "merge service" - it was up in the air as to whether it would stay or go in this cycle - I think the decision was "stay" so I do need to go back and rewrite the intro section and add info on that service. If it ends up going in this cycle I will have to redo that Lab.

Reply:
Yes, some docs on this new service will need to be added.